modules/navigation.js

```diff
@@ -0,0 +1,42 @@
+const year = document.querySelector('#year');
+const contact = document.querySelector('#contact');
+const bookLibrary = document.querySelector('#book-library');
+const newBook = document.querySelector('#new-book');
+const list = document.querySelector('.list');
+const add = document.querySelector('.add');
+const contactMe = document.querySelector('.contact-me');
+
+const dateArr = {
+  weekday: 'long',
+  month: 'long',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+  second: 'numeric',
+  hour12: 'false',
+};
+const today = new Date();
+const locale = navigator.language;
+
+year.textContent = today.toLocaleDateString('en-US', dateArr, locale);
+
+list.addEventListener('click', (e) => {
+  e.preventDefault();
+  bookLibrary.style.display = 'block';
+  contact.style.display = 'none';
+  newBook.style.display = 'none';
+});
+
```

Comment:
- [ ] From the project requirements, you are expected to use Lixon library for showing the date and time. Please take a look at the screenshot 👇 
![image](https://user-images.githubusercontent.com/90222110/199101544-ad12f403-6a7c-4cce-826e-894082f34808.png)
Please make sure you use it 😃 👍

Reply:
Hi, @BertrandConxy Thanks for the feedback, 
I mistakenly skipped using the luxon library to get the date and time,
I'll display the current date using the library